sampledb/templates/qr_template.tex

```diff
@@ -19,13 +19,12 @@
 \graphicspath{\VAR{gpath}}
 
 %\LabelGridtrue
-\numberoflabels=\LabelCols
+\numberoflabels=1
 \begin{document}
 \begin{labels}
 \BLOCK{for qr, uid in qrs}
-    \centering
-        \includegraphics[width=\VAR{width}, height=\VAR{height}, keepaspectratio]{\VAR{qr}}\\
-    \captionof*{figure}{\VAR{uid}}
+        \hfill\includegraphics[width=\VAR{width}, height=\VAR{height}, keepaspectratio]{\VAR{qr}}\hfill\includegraphics[width=\VAR{width}, height=\VAR{height}, keepaspectratio]{\VAR{qr}}\hfill\includegraphics[width=\VAR{width}, height=\VAR{height}, keepaspectratio]{\VAR{qr}}\hfill\\
```

Comment:
There is probably an extra tab here. We probably want the same indentation as the next line.